### PR TITLE
Clean backend

### PR DESCRIPTION
**Pull request by mabmalia**
Deleted a file called "hello.txt" (?)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,81 +1 @@
...
-### Starting the backend server
-Open the root folder in IntelliJ and import all the gradle dependencies (this has to be done only once of course).
- 
-Make sure that the database is running (see steps above) otherwise the backend won't have access to it. Then run the main class at `src/main/java/se/kth/sda6/skeleton/SkeletonApplication.java` to start the web server.
-
-### Starting the frontend development server
-To install the project dependencies for the frontend, open the terminal and make sure that the current directory is `frontend`. You can then run `npm install` to install all the dependencies needed for the project (This has to be done only once).
-
-To start the frontend server run `npm start` from the `frontend` directory. Make sure that database and backend server are also running so that the frontend can interact with the backend.
-When working on the frontend, open the `frontend` folder in your favorite Javascript IDE.
```
Comment (MeriM-dev) [inline README.md]: We can keep this part just as a reminder on how-to set it all up

File: README.md [hunk 2]
```diff
@@ -1,81 +1 @@
...
-with you on your repo. Go to your repo -> go to the `Settings` tab -> go to the `Collaborators` tab and then add your
-group members via their GitHub username. Now they will be able to push changes as well after they clone your repo.
-
-## FAQ
-### How can I connect to running database from terminal?
-Sometimes you might want to inspect the tables, run raw queries, seed the database, check that a certain backend action has been performed correctly. To do that you can simply run the following command
-
-`docker run -it --network host postgres:11-alpine psql -h localhost -U skeleton_user -p 5433 -W skeleton`
-
-Use the password specified in `docker-compose.yaml`.
```
Comment (MeriM-dev) [inline README.md]: This part here looks useful as well. We could keep this too.

**Review by MeriM-dev — APPROVED** [review]

**Comment by krismutalapova** [comment]
LGTM

File: README.md (hunk 1 above)
Comment (mabmalia) [reply README.md]: Done.

File: README.md (hunk 2 above)
Comment (mabmalia) [reply README.md]: Done.

**Review by elmenqui — APPROVED** [review]
LGTM - Have seen it working in Kris's computer

**Pull request merged**